apps/sms/views/inbox/js/inbox.js

```diff
@@ -163,8 +164,7 @@ var InboxView = {
     var threadId = node.dataset.threadId;
     var draftId = node.dataset.draftId;
 
-    var threadOrDraft = draftId ?
-      Drafts.byDraftId(+draftId) : Threads.get(+threadId);
+    var threadOrDraft = Threads.get(+(draftId || threadId));
```

Comment:
hmm, so are we going to unity the `thread` properties for all the cases in this patch? Not sure if it's confusing that `thread` we get doesn't have no draft property but it's draft actually :) But I think the original structure is also confusing...